packages/tupaia-web/src/features/Dashboard/ExportDashboard.tsx

```diff
@@ -82,7 +82,7 @@ export const ExportDashboard = ({ isOpen, onClose, dashboardItems }: ExportDashb
       projectCode,
       entityCode,
       dashboardName,
-      selectedDashboardItems: selectedItems.map(({ code }) => code),
+      selectedDashboardItems: selectedItems.map(({ code }) => code!),
```

Comment:
I think it was to handle a TS error that popped up which would break the build

Reply:
Ah I see now. Looks like we changed the `ListItemProps` so that `code` could be `undefined`.

This assertion would be safe, but maybe we could also try making the `CheckboxList` component be generic, so that it can handle any `ListItem` types as long as they extend `ListItemProps`? I might just do that in a separate PR to avoid holding this one up any longer 👍